mappings/net/minecraft/server/config/BannedIpsEntry.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS ww net/minecraft/server/config/BannedIpEntry
+CLASS ww net/minecraft/server/config/BannedIpsEntry
```

Comment:
Seems that this class only holds one IP. Does `BannedIpEntry` not make more sense?

Reply:
I see this. Should the list classes like BannedIpsList also be renamed to BannedIpList?

Reply:
I think that makes more sense, yeah.